webpack.dev.js

```diff
@@ -21,4 +23,21 @@ module.exports = merge(common, {
             }
         ]
     },
+    plugins: [
+        new HtmlWebpackPlugin({
+            filename: 'index.html',
+            template: 'index.html'
+        }),
+        new ConcatPlugin({
+            name: 'scripts/apploader.js',
+            filesToConcat: ['./scripts/apploader.js']
+        })
+    ],
+    devServer: {
+        proxy: {
+            "!/**/*.html" : "http://localhost:8096",
+            "!/**/*.css" : "http://localhost:8086",
+            "!/**/*.js" : "http://localhost:8086"
```

Comment:
We have images and other assets that need to get proxied as well, would it be insecure to just pass everything in the source folder?

Reply:
I don't think that's possible with glob-patterns (that's what webpack-dev-server uses here). I think we can just add those file patterns on demand if it starts to be a problem.

Reply:
Actually, this proxy shouldn't even be required. If you add the following line to the index file the web client doesn't need to proxy any requests.

```
<script>window.appMode='standalone';</script>
```

Reply:
Yes, that brings up a server selection dialog. If i put it in there, this will always be shown, also in the production bundle. Is that okay for you?

Reply:
Nevermind, i added `standalone.js`which will be in injected in dev mode.